src/pal/inc/pal.h

```diff
@@ -5466,7 +5466,14 @@ PALIMPORT double __cdecl _copysign(double, double);
 extern "C++" {
 inline float fabsf(float _X)
 {
-    return ((float)fabs((double)_X)); }
+    return ((float)fabs((double)_X));
+}
+
+#ifdef BIT64
+inline __int64 abs(__int64 _X) {
+    return labs(_X);
```

Comment:
Oh, right, but then we are completely missing an implementation of abs(__int64). Should we call the C version llabs then?